src/generate_mhq_sample/20_mhq_terr_sample_admin.Rmd

```diff
@@ -241,35 +278,42 @@ check_plots_data_import %>%
 
 ### Determine type in plot center and check if the observed type corresponds with the target type
 
-Vegetation type cover is measured both in the square plot and the circle plot. The field protocol prescribes that a plot is only recorded when the plot center is located within (one of) the target habitat type(s). When the observed type is different from (one of) the target type(s), the plot can be recorded when the GRTS ranking of the sample location is smaller than the largest GRTS ranking in the set of samples for the observed type. 
+Vegetation type cover is measured both in the square plot and the circle plot. 
+The field protocol prescribes that a plot is only recorded when the plot center is located within (one of) the target habitat type(s). 
+When the observed type is different from (one of) the target type(s), the plot can be recorded when the GRTS ranking of the sample location is smaller than the largest GRTS ranking in the set of samples for the observed type. 
 
 In some cases the subtype is not specified. When this is the case, we assume that the subtype of the observed type corresponds to the target type.
 
-Some recorded plots contain different habitat types or other cover types. However it is not explicitely recorded which of the cover types is located in the center of the plot. Yet this is essential information to determine if a sample is valid. Therefore, when a plot contains different cover types we assume that the plot center is located in the type recorded in the square plot that corresponds to (one of) the target type(s). When none of the target habitat types are observed we assume that the plot center is located in the type that belongs to the same main type as (one of) the target habitat types. We apply these rules and check of we select one type for each plot this way.
+Some recorded plots contain different habitat types or other cover types. 
+However it is not explicitely recorded which of the cover types is located in the center of the plot. 
+Yet this is essential information to determine if a sample is valid. 
+Therefore, when a plot contains different cover types we assume that the plot center is located in the type recorded in the square plot that corresponds to (one of) the target type(s). 
+When none of the target habitat types are observed we assume that the plot center is located in the type that belongs to the same main type as (one of) the target habitat types. We apply these rules and check of we select one type for each plot this way.
```

Comment:
For example: the target types in the reference point are 4010 and 6510_hu. the observed types in the plot are 6510_hus and 6210. then we assume that 6510_hus is the observed type in the reference point as it belongs to the main type 6510, which is the main type of one of the target types.

Reply:
OK, I see what you mean now. The confusion came from "... the type that belongs to the same main type as ...". From "When none of the target habitat types are observed" it is not evident that _another subtype from a same_ (targetted) main type _is_ observed, which is the case you handle. Best describe the handled circumstance more clearly.

Do you describe what happens in the following cases then (perhaps they don't occur?), where 4010 and 6510_hu are target types?

1. 6210 and 6410_mo were observed 
2. 4010 and 6510_hu (the target types) were observed

Reply:
OK. I clarified it by using an example. 
Unfortunately, the two other cases you describe do occur (only a couple of cases). In case of 1. we select the type with the highest cover. In case of 2. we report it to the fieldwork coordinator to clarify.

Reply:
OK, best distinguish between those cases in the text

Reply:
OK I did